CMakeLists.txt

```
@@ -38,6 +38,7 @@ add_library(PortAudio
 target_include_directories(PortAudio PUBLIC
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
   $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src/common>
+  $<INSTALL_INTERFACE:include>
```

```suggestion
  $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
```
Thanks for catching this. I think it would be better to use the GNUInstallDirs variable than hardcoding `include`.